### PR TITLE
Added additional files to github workflow

### PR DESCRIPTION
**Pull request by brettdavis4**

### CODE REVIEW

**Review by korymarks — APPROVED** [review]
LGTM

File: .github/workflows/code-checks.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Code Checks
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (dbirks) [inline .github/workflows/code-checks.yml]: This project still uses the `master` branch, can you change these references throughout? We haven't gotten to standardizing on the main branch yet.

File: .github/workflows/publish-package.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+name: Publish to Github Packages
```
Comment (dbirks) [inline .github/workflows/publish-package.yml]: This publish-package.yml file duplicates what's done in [main.yml](https://github.com/polarislabs/pg-to-dbml/blob/develop/.github/workflows/main.yml). 
Can you maybe delete this one, and rename the existing main.yml to publish-package.yml?

File: .github/workflows/code-checks.yml (hunk 1 above)
Comment (camoore1979) [inline .github/workflows/code-checks.yml]: `pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```

File: .github/workflows/code-checks.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    paths:
+      - "src/**"
+      - "**.js"
+      - "**.json"
+  push:
+    branches:
+      - main
```
Comment (camoore1979) [inline .github/workflows/code-checks.yml]: `pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```

**Review by dbirks — CHANGES_REQUESTED** [review]

File: .github/workflows/prod-auto-tag-and-release.yml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+name: Auto Tag and Release
+
+on:
+  push:
+    branches:
+      - main
```
Comment (camoore1979) [inline .github/workflows/prod-auto-tag-and-release.yml]: `pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```

File: .github/workflows/prod-check-package-version.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+name: 'Check package.json version'
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (camoore1979) [inline .github/workflows/prod-check-package-version.yml]: `pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```

File: .github/workflows/publish-package.yml [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+on:
+  push:
+    branches:
+      - main
```
Comment (camoore1979) [inline .github/workflows/publish-package.yml]: `pg-to-dbml` was prior to the Github rollout of the new main branch name.

```suggestion
      - master
```

**Review by camoore1979 — CHANGES_REQUESTED** [review]
Small catch, see below.

You will need to doublecheck other repos. One repo (I forget which one) has a `main` branch, all others have `master`. Make sure and align the scripts to that.

**Review by dbirks — APPROVED** [review]
Looks good 👍

**Pull request merged**